### ISSUE TITLE
deprecate xml-signature link relationship

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#678]
Without refinement chains, this property doesn't appear to have a use. It used to indicate the signature that was issued by the authority defined in a meta-auth property, but it's not possible to link them anymore and I'm not aware these properties were used in practice.

**Comment by murata2makoto** [comment]
I think that we need a mechanism different from @refines for digital signatures eventually.  (Arguably, not in 3.1, but I do not want to drop digital signatures from OCF.

EU is going to adopt [XAdES ](https://en.wikipedia.org/wiki/XAdES) as EN.  I recently attended a meeting of ETSI, which standardizes XAdES.  People do not think XAdES for EPUB is useful for libraries or publishers.  But I said that manufactures might want to create XAdES digital signatures for their manuals in EPUB., and they did not oppose to this possibility.  Still an imaginary use case, though.

**Comment by mgylling** [comment]
Proposed resolution: do not provide a (new) way for OPF to identify xml signatures for resources. Another generic mechanism needs to put in place in the future, in those cases where OCF's signatures.xml does not suffice.

**Comment by murata2makoto** [comment]
Agreed since @refines is still usable.

**Comment by mattgarrish** [comment]
Looking at this again, I think my problem is that the relationship was reversed as I was puzzling the example in the spec. It could contain a signature for the epub publication, but in the examples it refines the metadata authority which refines an ISBN. Given such a convoluted chain, how does one know whether it is the publication's signature or something else?

It would make sense without refines as purely a means of indicating which signature is for the publication, or if refines pointed at a resource instead of other metadata.

In other words, sounds good to me as a resolution, but I don't really care that much anymore since we might have wound up making it more sensible by removing refinements.

**Comment by mattgarrish** [comment]
Updated the link vocabulary to deprecate xml-signature and note that the problem may be returned to in a future version:
https://idpf.github.io/epub-vocabs/package/link/#xml-signature

Please close this issue if no additional changes are required.
